### PR TITLE
test(csharp): inline parameters

### PR DESCRIPTION
**Pull request by morganleroi**
## 🧭 What and Why

🎟 JIRA Ticket: DI-1716

### Changes included:

Inline parameters for CTS and Snippets.

### CODE REVIEW

File: scripts/formatter.ts [hunk 1]
```diff
@@ -5,7 +5,11 @@ export async function formatter(language: string, cwd: string): Promise<void> {
   const spinner = createSpinner(`running formatter for '${language}' in '${cwd}'`);
   switch (language) {
     case 'csharp':
-      await run('dotnet format', { cwd, language });
+      if (cwd.includes('tests')) {
```
Comment (shortcuts) [inline scripts/formatter.ts:8]: this needs `snippets` now too

File: tests/output/csharp/.editorconfig
Comment (shortcuts) [inline tests/output/csharp/.editorconfig:1]: can you apply this to the snippets too?

**Review by shortcuts — COMMENTED** [review]
❤️

File: scripts/formatter.ts (hunk 1 above)
Comment (morganleroi) [reply scripts/formatter.ts:8]: Yes ok ! I'll try to generate snippet and take a look at the result

File: scripts/formatter.ts [hunk 2]
```diff
@@ -5,7 +5,11 @@ export async function formatter(language: string, cwd: string): Promise<void> {
   const spinner = createSpinner(`running formatter for '${language}' in '${cwd}'`);
   switch (language) {
     case 'csharp':
-      await run('dotnet format', { cwd, language });
+      if (cwd.includes('tests') || cwd.includes('snippets')) {
```
Comment (millotp) [inline scripts/formatter.ts:8]: why not use csharpier for the client too ?

**Review by millotp — COMMENTED** [review]
looks way better gg !

**Pull request merged**